src/polyit_rage/websocket_clients/client_manager.py

```
@@ -0,0 +1,406 @@
+            status.last_check = datetime.now(timezone.utc)
+            status.is_healthy = client.is_connected
+            if client.is_connected:
+                status.error_message = None
+    
+    async def _health_check_loop(self) -> None:
+        """Periodic health check for all clients."""
+        while self._running:
+            try:
+                await asyncio.sleep(self._health_check_interval)
+                
+                for name, client in self._clients.items():
```

Ruff (0.14.11)

Loop control variable `client` not used within loop body

Rename unused `client` to `_client`

(B007)